bin/elixir.ps1

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env pwsh
+
+$MinPowerShellVersion = [version]"7.2.0"
+
+if ($MinPowerShellVersion.CompareTo([version]$PSVersionTable.PSVersion) -eq 1) {
+  Write-Error "This script requires PowerShell version 7.2 or above. Running on $($PSVersionTable.PSVersion)"
+}
+
+$ELIXIR_VERSION = "1.17.0-dev"
+$ScriptPath = Split-Path -Parent $PSCommandPath
```

Comment:
In my experience, variables are camelCase and Parameters are PascalCase.  There is no rule in the official guides about variables being camelCase, whereas there is one about Parameters being PascalCase.